packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,353 @@ Specific property types support a subset of [JSON Schema validations](https://to
 | `number`      | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with double-precision 64-bit IEEE 754 floating point number constraint | `minimum` and `maximum`     |
 | `integer`     | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with 32-bit signed two's complement integer constraint           | `minimum` and `maximum`     |
 | `array`       | [JSON Array](https://tools.ietf.org/html/rfc7159#section-5)                                                                         |                             |
+
+## Unique attribute
+
+<ApiLifecycle access="ea" />
+
+You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as an employee identification number. You can declare a maximum of five unique properties for each user type and five unique properties in the Okta group profile. Different user types can have the same or different unique properties (up to the limit of five per type).
+
+Unique properties in the Okta user profile share a single namespace across all [user types](/docs/reference/api/user-types) in an org. If user types A and B both contain the property `ice cream` and you identify it as unique in both profiles, then if user type A has the value `chocolate`, no other users of type A or B (or any other user type with `ice cream` declared unique) can have that value.
+
+Properties that aren't unique aren't tracked for uniqueness. If the property `candy` is unique in type E and not unique in type F, and a user of type E has the value `caramel` for the property, then no other users of type E can have the value `caramel` for the property. But any number of users of type F can have the value `caramel`. Although `candy` is unique in E, it isn't unique in F, so the value for the property in users of type F doesn't matter.
```

Comment:
How about this:

> Properties that aren't unique also aren't tracked for uniqueness. Suppose the property `candy` is unique in type E and not unique in type F. If a user of type E has the value `caramel` for the `candy` property, no other users of type E can also have the value `caramel`. But any number of users of type F can subsequently be assigned the value `caramel`. Furthermore, because `candy` is not unique in F, any values users of type F may have are not considered when enforcing uniqueness for users of type E.  No matter how many users of type F already have the value `cotton`, it may be assigned to a user of type E as long as no other such user already has that value.

Reply:
I think Dayton's version elaborates better 😇 . Updated.